rasa/core/policies/ted_policy.py

```diff
@@ -805,7 +805,7 @@ def _pick_confidence(
         )
         return confidences[0], self.only_e2e
 
-    def predict_action_probabilities(
+    def predict_action_probabilities(  # type: ignore[override]
```

Comment:
what about defining `precomputations` as a kwargs only optional argument on the parent class method ? (same for `train()`)

Reply:
That would need amending the signatures of the same method for `MemoizationPolicy` and `RulePolicy` who don't make use of this parameter either. I will reverse the order of the kwargs, so that `precomputations` appears after `rule_only_data`, see if that causes anything to fail.